common/src/main/java/dev/schmarrn/lighty/event/KeyBind.java

```diff
@@ -48,9 +48,6 @@ public static void handleKeyBind(Minecraft client) {
         if (newToggleState && !KeyBind.oldToggleState) {
             ModeLoader.toggle();
         }
-        if (newKeyState && !KeyBind.oldKeyState) {
```

Comment:
Being able to change the Lighty Settings is pretty important, please revert this removal.

Reply:
Sorry about that. Because of how I implemented the integration in ModMenuIntegration.java, I needed to change the constructor for ModeSwitcherScreen(), which caused an error with the keybind. At the time, I removed the keybind to get rid of the error, thinking that ModMenu would be a good enough substituion.

However, you're right in being able to change the settings being important, since some might not install ModMenu with it.

I'll add a second constructor to fix this. Doing so should restore the original keybind, while keeping integration with ModMenu.